src/components/PlantsV2/components/HectaresPlantedCard.tsx

```diff
@@ -0,0 +1,93 @@
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import { Box, LinearProgress, Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+import React, { useMemo } from 'react';
+import { useAppSelector } from 'src/redux/store';
+import { selectPlantingSite } from 'src/redux/features/tracking/trackingSelectors';
+import { ObservationResults } from 'src/types/Observations';
+import { getShortDate } from 'src/utils/dateFormatter';
+import { useLocalization } from 'src/providers';
+
+type HectaresPlantedCardProps = {
+  plantingSiteId: number;
+  observation?: ObservationResults;
+};
+
+export default function HectaresPlantedCard({ plantingSiteId, observation }: HectaresPlantedCardProps): JSX.Element {
+  const theme = useTheme();
+  const plantingSite = useAppSelector((state) => selectPlantingSite(state, plantingSiteId));
+  const locale = useLocalization();
+
+  const totalArea = plantingSite?.areaHa ?? 0;
+  const totalPlantedArea = useMemo(() => {
+    return (
+      plantingSite?.plantingZones
+        ?.flatMap((zone) => zone.plantingSubzones)
+        ?.reduce((prev, curr) => (curr.plantingCompleted ? +curr.areaHa + prev : prev), 0) ?? 0
+    );
+  }, [plantingSite]);
+  const percentagePlanted = totalArea > 0 ? Math.round((totalPlantedArea / totalArea) * 100) : 0;
+
+  return (
+    <OverviewItemCard
+      isEditable={false}
+      contents={
+        <Box display='flex' flexDirection='column'>
+          <Typography fontSize='16px' fontWeight={600} marginBottom={theme.spacing(5)}>
+            {observation?.completedTime
+              ? strings.formatString(
+                  strings.HECTARES_PLANTED_CARD_TITLE,
+                  getShortDate(observation.completedTime, locale.activeLocale)
+                )
+              : ''}
+          </Typography>
+          <Box display={'flex'} alignItems='baseline'>
+            <Typography fontSize='84px' fontWeight={600} lineHeight={1} marginBottom={theme.spacing(3)}>
+              {totalPlantedArea}
```

Comment:
may need to localize this, create a ticket to follow up later? "20,000" instead of "20000" (and "20.000" for esp, etc.).
Unless the BE returns localized values (my understanding is that search results alone are localized).
There is a number formatter under utils/useNumber.ts
I suppose my suggestion applies to all charts.. cc @charlie-tf 
A one-time refactor later may work.

Reply:
Thanks for the heads up!